content/docs/cloud/concepts.md

```diff
@@ -9,7 +9,7 @@ A Neon user is identified by their email address.
 
 A user registers and authenticates in Neon Web UI with their GitHub or Google account. More authentication methods are coming soon.
 
-Once authenticated, a user can create and access [Projects](#project) and [query Project data](../tutorials#query-via-ui). You can also manage [Postgres Users](#postgres-users) and [Databases](#postgres-databases) in each Project.
+Once authenticated, a user can create and access [Projects](#project) and [query Project data](../get-started-with-neon/tutorials#query-via-ui). You can also manage [Postgres Users](#postgres-users) and [Databases](#postgres-databases) in each Project.
```

Comment:
is this page available to users from the sidebar?
i can access it [directly](https://websitemain62807-redirects.gtsb.io/docs/cloud/concepts/), but i don't see how a user can discover it.

Reply:
This page is now deprecated, each section was moved to a different page and either expanded on or rewritten.

Reply:
> This page is now deprecated

Sorry, I didn't realize this.

> each section was moved to a different page

I couldn't find where the following sections were moved to: Neon User, Project, Postgres Users, Postgres Databases

Also the 'Compute Config' section does not belog to 'Technical Preview Free Tier'

Sorry I haven't noticed this earlier.

Reply:
Let's move the text of the Compute Config section to the Compute Lyfecycle page.

Reply:
Hi @stepashka just updating you that Jabali update the link and move the branches to compute config. Can we merge this?